src/v/rpc/connection_cache.cc

```diff
@@ -20,7 +20,18 @@ namespace rpc {
 connection_cache::connection_cache(
   ss::sharded<ss::abort_source>& as,
   std::optional<connection_cache_label> label)
-  : _label(std::move(label)) {
+  : _max_connections(8)
```

Comment:
I think I should probably just remove this constructor entirely. I added it in mainly to avoid having to change a bunch of unit tests. But it's kinda a foot-gun now that I look at it again.